Common/GameSystem/PackOpening/CardPools.cs

```diff
@@ -15,10 +15,9 @@ namespace TerraTCG.Common.GameSystem.PackOpening
 	internal class CardPools
 	{
 		// Cards that can appear in any pack, generally, "not good" cards
-		public static CardCollection AllCards => new()
+		public static CardCollection CollectableCards => new()
```

Comment:
```suggestion
		public static CardCollection AllCards => new()
```

既存の命名を変える必要はないように思います。